query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -419,6 +419,19 @@ public void testAttributeExpressionQueries(
     assertEquals(expectedValue, rows.get(0).getColumn(0).getString());
   }
 
+  @Test
+  public void testNotContainsKeyQuery() throws IOException {
+    LOG.info("Not contains key query");
+    Iterator<ResultSetChunk> itr =
+        queryServiceClient.executeQuery(
+            buildQueryFromJsonFile("query4.json"), TENANT_ID_MAP, 10000);
```

Comment:
is there a 1-1 mapping between the test method and query file, if yes then maybe you can name the query file after the method name?